src/Pelagos/Util/Search.php

```diff
@@ -35,28 +35,38 @@ class Search
     /**
      * Elastic index title field.
      */
-    const INDEX_FIELD_TITLE = 'title';
+    const TITLE = 'title';
 
     /**
      * Elastic index abstract field.
      */
-    const INDEX_FIELD_ABSTRACT = 'abstract';
+    const ABSTRACT = 'abstract';
 
     /**
      * Elastic index dataset submission authors field.
      */
-    const INDEX_FIELD_DS_AUTHOR = 'datasetSubmission.authors';
+    const DS_AUTHORS = 'datasetSubmission.authors';
 
     /**
      * Elastic index dataset submission theme keywords field.
      */
-    const INDEX_FIELD_DS_THEME_KEYWORDS = 'datasetSubmission.themeKeywords';
+    const DS_THEME_KEYWORDS = 'datasetSubmission.themeKeywords';
+
+    /**
+     * Elastic index mapping.
+     */
+    const ELASTIC_INDEX_MAPPING = array(
```

Comment:
You don't really need those other constants now do you?

Reply:
Yeah, i think so. removed the individual ones